sample_data/events.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "_id": "6437907585008f171cf2994c",
+    "recurring": true,
+    "recurrance": "ONCE",
```

Comment:
Make all the events recurring. That way people will always see the events in the default calendar views which would be the current day, month, week.